iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/infospace/truncator/factory/TestDocumentMetadataAvroTruncatorFactory.java

```diff
@@ -0,0 +1,17 @@
+package eu.dnetlib.iis.wf.importer.infospace.truncator.factory;
+
+import eu.dnetlib.iis.wf.importer.infospace.truncator.DocumentMetadataAvroTruncator;
+
+public class TestDocumentMetadataAvroTruncatorFactory implements DocumentMetadataAvroTruncatorFactory {
```

Comment:
OK, so the truncators solution, apart from being a subject for modular testing, is kind of a wrapper encapsulating the thresholds for 6 dimensions, right? 

So instead of overwriting 6 dimensions for the test we can switch the truncator implementation. It's nice, on one hand because we can keep the workflow definition compact, but on the other hand when having 6 overridable threshold parameters we could make them easily overridable when it turns out our pick statically defined in `ImportInformationSpaceConstants` was not accurate. It might be a matter of a personal opinion but I would prefer to have more flexibility in this regard.

Reply:
After changing the way of passing truncation parameters to importer job this class is no longer needed. Truncation parameters are now passed as dynamic parameters with default values. For tests the values are explicitly provided and there is no need for any 'factory' classes.